test/built-ins/Temporal/PlainDateTime/prototype/toString/calendarname-auto.js

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.tostring
+description: Possibly display calendar when calendarName is "auto"
+features: [Temporal]
+---*/
+
+
+const dt = new Temporal.PlainDateTime(1976, 11, 18, 15, 23);
+const cal = {
+  toString() { return "bogus"; }
+};
+const expected = "1976-11-18T15:23:00";
+
+assert.sameValue(dt.toString(), expected, "default is calendar = auto (zero arguments)");
+assert.sameValue(dt.toString({ calendarName: "auto" }), expected, "shows only non-ISO calendar if calendarName = auto");
+
+assert.sameValue(
+  dt.withCalendar("iso8601").toString({ calendarName: "auto" }),
```

Comment:
This doesn't really add anything over testing with `dt` directly. I would like to see a check using a non-builtin calendar whose `toString` returns `"iso8601"` instead.

Reply:
Good idea -- done.